electronicparsers/openmx/parser.py

```diff
@@ -717,6 +734,27 @@ def parse(self, mainfile: str, archive: EntryArchive, logger):
             name='OpenMX', version=str(mainfile_parser.get('program_version'))
         )
 
+        stress_tensor_enabled = mainfile_parser.get('scf.stress.tensor')
+        if stress_tensor_enabled == "on":
+            mainfile_dir = path.dirname(mainfile)
+            stdout_path = None
+            for file in listdir(mainfile_dir):
+                f_path = path.join(mainfile_dir, file)
+                if path.isfile(f_path):
+                    with open(f_path) as input_file:
+                        for index, line in enumerate(input_file):
+                            if r'Welcome to OpenMX' in line:
+                                stdout_path = f_path
+                                break
+                            if index == 500:
+                                break
+                if stdout_path is not None:
+                    break
```

Comment:
There can be in theory more calculations in the directory and hence more stdout files (but I believe this should not be too common). So in theory it would be best to do the Uele consistency check here, so we can look for other files if we fail it. However, I don't want to overengineer this (and we also don't have the other parsing done at this point I think), so if it would complicate things too much, just leave it as is.